app/src/main/java/com/github/h3lp3rs/h3lp/forum/ForumAnswersActivity.kt

```diff
@@ -0,0 +1,78 @@
+package com.github.h3lp3rs.h3lp.forum
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.github.h3lp3rs.h3lp.R
+import com.github.h3lp3rs.h3lp.forum.ForumPostsActivity.Companion.selectedPost
+import com.github.h3lp3rs.h3lp.forum.data.ForumPostData
+import com.github.h3lp3rs.h3lp.signin.SignInActivity.Companion.getName
+import com.xwray.groupie.GroupAdapter
+import com.xwray.groupie.Item
+import com.xwray.groupie.ViewHolder
+import kotlinx.android.synthetic.main.activity_forum_answers.*
+import kotlinx.android.synthetic.main.answer_forum_row.view.*
+
+/**
+ * Activity where the user sees all answers of the selected post and enters an answer
+ */
+class ForumAnswersActivity : AppCompatActivity() {
+    private val adapter = GroupAdapter<ViewHolder>()
+    private lateinit var category : String
+    private lateinit var forum : Forum
+
+    override fun onCreate(savedInstanceState : Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_forum_answers)
+
+        recycler_view_forum_answers.adapter = adapter
+        category = selectedPost.post.category.name
+
+        forum = ForumCategory.categoriesMap[category]?.let { ForumCategory.forumOf(it) }!!
```

Comment:
FYI: Here You will simply replace CachedForum(oldForum) when implementing the cache, so nothing dramatic ;)

Reply:
Don't do it now though, still needs more robust testing before deployment